### ISSUE TITLE
Links are very, very dark

### ISSUE THREAD

**Issue by pluiedev** [ghostty-org/website#42]
A somewhat ironic screenshot demonstrating the problem:

<img src="https://github.com/user-attachments/assets/2c3da284-b163-4c5b-8d08-dd43ab5352aa" height=200>

And they also fail WCAG's small text contrast requirements (courtesy of @matthew-hre):

<img src="https://github.com/user-attachments/assets/d1b1068b-76ee-468f-ac50-7931e9ae4f82" height=400>
